examples/resources/rule_aws_sqs.tf

```diff
@@ -5,15 +5,15 @@ resource "ably_rule_sqs" "rule0" {
     channel_filter = "^my-channel.*",
     type           = "channel.message"
   }
-  aws_authentication = {
-    mode     = "assumeRole",
-    role_arn = "cccc"
-  }
   target = {
     region         = "us-west-1",
     aws_account_id = "123456789012",
     queue_name     = "aaaaaa",
     enveloped      = false,
     format         = "json"
+    aws_authentication = {
```

Comment:
This one should be `authentication` rather than `aws_authentication`

Reply:
Thanks. Had to miss one.